### DISCUSSION TITLE
US: Add the user story title here

### DISCUSSION THREAD

**Discussion by jmark20** [eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework#180] [General]
### Description
add the user story description here

### User Story
**User**: add the user profile/persona here
**Goal**: add the goal of the user story
**Reason**: add the reason why this user story has been added

### Acceptance Criteria
The following scenarios shall be covered.

_Scenario 1_: add the scenario title here

Add the scenario description here

**Comment by digeorgi** [comment]
Issue has been converted to a Discussion to enable the discussion on the procedural aspects of proposing an enhancement via the creation of a User Story.
